.github/dependabot.yml

```diff
@@ -17,6 +17,7 @@ updates:
         patterns:
           - "xunit.*"
           - "Microsoft.NET.Test.Sdk"
+          - "Microsoft.Testing.*"
           - "coverlet.*"
           - "GitHubActionsTestLogger"
```

Comment:
I think you removed the GH action logger, isn't it?

Reply:
Yup. This is just dependabot configuration so I'm leaving it there just in case it's added back after it supports MTP.